helm/patroni/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+name: patroni
```

Comment:
can we fork and send these changes back upstream?

Reply:
maybe some of them. I did delete the Endpoint object as it seemed to be causing an issue

Reply:
That could be solved by making the Endpoint should togglable, no?